testing/simapp/app.go

```diff
@@ -128,6 +128,7 @@ import (
 	ibcexported "github.com/cosmos/ibc-go/v8/modules/core/exported"
```

Comment:
I did these changes so that I could write tests, but I am aware that we will refactor this once 07-celestia has its own go.mod.